tgui/packages/tgui/interfaces/SecurityRecords/helpers.tsx

```diff
@@ -0,0 +1,16 @@
+import type { GenericRecord } from './types';
+
+/** Matches search by name or rank */
+export const isRecordMatch = (record: GenericRecord, search: string) => {
+  if (!search) return true;
+  const { name, rank } = record;
+
+  switch (true) {
```

Comment:
I'm not sure a switch case statement is really the way to do this

Reply:
Changed to if statements.